src/btree/bt_import.c

```diff
@@ -98,13 +100,16 @@ __wt_import_repair(WT_SESSION_IMPL *session, const char *uri, char **configp)
      */
     cfg[1] = a->data;
     cfg[2] = checkpoint_list;
-    cfg[3] = "checkpoint_backup_info=";
+    WT_ERR(__wt_reset_blkmod(session, a->data, buf));
+    cfg[3] = buf->mem;
     cfg[4] = "checkpoint_lsn=";
     WT_WITH_SCHEMA_LOCK(session,
       ret = __wt_snprintf(fileid, sizeof(fileid), "id=%" PRIu32, ++S2C(session)->next_file_id));
     WT_ERR(ret);
     cfg[5] = fileid;
     WT_ERR(__wt_config_collapse(session, cfg, &config_tmp));
+    __wt_verbose(
+      session, WT_VERB_BACKUP, "printing after config (import repair) %s\n", (char *)config_tmp);
```

Comment:
If we want to keep this verbose message then change the text to something like "Import checkpoint configuration %s"
And you don't need/want the \n as the verbose system already adds them.

Reply:
removed! was planning to remove once PR is ready to merge